stream-chat-android-ui-components/api/stream-chat-android-ui-components.api

```diff
@@ -653,6 +653,7 @@ public final class io/getstream/chat/android/ui/ChatUI {
 	public static final fun getMimeTypeIconProvider ()Lio/getstream/chat/android/ui/MimeTypeIconProvider;
 	public static final fun getNavigator ()Lio/getstream/chat/android/ui/common/navigation/ChatNavigator;
 	public static final fun getQuotedAttachmentFactoryManager ()Lio/getstream/chat/android/ui/message/list/adapter/viewholder/attachment/QuotedAttachmentFactoryManager;
+	public static final fun getShowDateSeparatorInEmptyThread ()Z
```

Comment:
We should write a changelog message  and add this to the [configuration docs](https://getstream.io/chat/docs/sdk/android/ui/general-customization/chatui/) before merging.